easybuild/easyconfigs/v/VCFtools/VCFtools-0.1.14-foss-2015b.eb

```diff
@@ -0,0 +1,41 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+# Author: Adam Huffman
+# adam.huffman@crick.ac.uk
+# The Francis Crick Institute
+
+easyblock = 'ConfigureMake'
+
+name = 'VCFtools'
+version = '0.1.14'
+
+homepage = "https://vcftools.github.io"
+description = """The aim of VCFtools is to provide 
+ easily accessible methods for working with complex 
+ genetic variation data in the form of VCF files."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+toolchainopts = {'optarch': True}
+
+source_urls = ['https://github.com/%(namelower)s/%(namelower)s/releases/download/v%(version)s/']
+sources = ['%(namelower)s-%(version)s.tar.gz']
+
+perl = 'Perl'
+perlver = '5.20.2'
+perlsuffix = '-bare'
+dependencies = [
+    (perl, perlver, perlsuffix),
```

Comment:
@verdurin: are you sure a bare Perl install is sufficient?

since it's a dependency, why not include the full Perl install (so this can be combined with other Perl tools that require non-standard Perl modules to be available)?

Also, why no versionsuffix anymore?